task.py

```diff
@@ -920,7 +928,7 @@ def __init__(self, ts: TestSettings, server: ServerTask):
         )
 
         pod_type = ts.client_pod_type
-        node_name = self.node_name
+        node_name = self.node_name_sanitized()
```

Comment:
please rename the variable `node_name` to `node_name_sanitized`, as this is what it is now.